### PR TITLE
[#623] Expose console using cert manager

### PR DESCRIPTION
**Pull request by gaohoward**

### CODE REVIEW

**Review by rvais — COMMENTED** [review]
So far I can't see anything that would be obviously wrong. It's a draft so I expect more changes.  Too much to keep everything in my head, especially since it's written in go-lang.

File: bundle/manifests/broker.amq.io_activemqartemises.yaml [hunk 1]
```diff
@@ -1970,8 +1974,6 @@ spec:
                         type: integer
                       grpc:
                         description: GRPC specifies an action involving a GRPC port.
```
Comment (rvais) [inline bundle/manifests/broker.amq.io_activemqartemises.yaml:1976]: What is this related to?

Comment (gaohoward) [reply bundle/manifests/broker.amq.io_activemqartemises.yaml:1976]: This new field is auto generated because of our dependency upgrade (which is needed by the cert manager package dependency)

**Comment by gaohoward** [comment]
I'm closing this as it's included in another PR https://github.com/artemiscloud/activemq-artemis-operator/pull/673
